hail/Makefile

```diff
@@ -271,9 +271,13 @@ set-docs-sha: deploy-annotation-db
 .PHONY: deploy
 deploy: tag
 
+.PHONY: install-dev-deps
+install-dev-deps:
+	$(PIP) install -U -r python/dev-requirements.txt
+
 .PHONY: install-deps
-install-deps:
-	$(PIP) install -U -r python/requirements.txt -r python/dev-requirements.txt
+install-deps: install-dev-deps
```

Comment:
This means that  `make install-deps` installs dev dependencies too, right? I feel like that's weird. I would expect `install-dev-deps` to install the regular deps, but I would not expect `install-deps` to install the dev ones.

Reply:
This keeps the behavior of install-deps consistent with the current master.